scripts/run-benchmarks.py

```diff
@@ -0,0 +1,185 @@
+import subprocess
+import json
+import os
+import re
+
+import click
+import cpuinfo
+
+BASE_REPO_URL = 'https://github.com/lumol-org/lumol.git'
+
+INDIVIDUAL_BENCH_TEMPLATE = """
+<details><summary>%s %s</summary>
+  <p>\n\n
+\n\n
+```bash\n
+%s\n\n
+```\n\n
+
+</p></details>
+"""
+
+COMPARISON_TEMPLATE = """
+<details><summary>%s %s</summary>
```

Comment:
Yes sure, it's indeed better for long string. I still find the older nicer for smaller strings (just because `%` is shorter that `.format()`), is it a problem if we use them both ?